web-client/src/components/ContactInfo.jsx

```diff
@@ -36,23 +36,42 @@ function Information(props) {
 
             '&:hover' :
             {
-                color: 'red',
+                color: 'grey',
             }
         }
     });
 
     const style = styles();
 
-    return (
+    
+    if(props.street === false)
+    {
+        return (
         <div className={style.main}>
             <div className={style.left}>
-                <img src={props.path} className={style.photo} alt="not found!"></img>
+                <img src={props.path} className={style.photo} alt={props.icon}></img>
```

Comment:
ok but I guess later we won't know that "icon" is for alt text (maybe different name?)